src/server/logger.js

```diff
@@ -1,27 +1,97 @@
 'use strict';
 
-var levels = ['trace', 'info', 'log', 'debug', 'warn', 'error'],
-    debug = require('debug');
+const chalk = require('chalk');
+
+const LEVELS = ['trace', 'info', 'log', 'debug', 'warn', 'error'],
+    LEVELCOLORS = {
+        'trace': {
+            bg: 'bgBlack',
```

Comment:
maybe no background/transparent instead of black?

Reply:
Default is now used instead